docs/guides/deps/building-acc-stratum-deps.md

```diff
@@ -1,4 +1,4 @@
-# Building Stratum Dependencies for the ACC
+# Building Dependencies for the ACC
```

Comment:
Should consider renaming the file too if we are dropping the language of "Stratum Dependencies"

Reply:
I'm not dropping it, I'm shortening it so it doesn't wrap in the sidebar. (It's a compromise, not a principle.) The problem may go away if I can figure out how to adjust the layout of the frame, in which case I'll change the title back.

Reply:
Translation: I don't want to edit everything "for consistency" right now and then have to change it back later when I've found a better solution.